src/braket/default_simulator/gate_operations.py

```diff
@@ -810,6 +810,36 @@ def _base_matrix(self) -> np.ndarray:
 def _cswap(instruction) -> CSwap:
     return CSwap([instruction.control, *instruction.targets])
 
+class PRx(GateOperation):
+    """
+    PhaseRx gate.
+    """
+
+    def __init__(self, targets, angle_1, angle_2, ctrl_modifiers=(), power=1):
+        super().__init__(
+            targets=targets,
+            ctrl_modifiers=ctrl_modifiers,
+            power=power,
+        )
+        self.angle_1 = angle_1
+        self.angle_2 = angle_2
+
+    @property
+    def _base_matrix(self) -> np.ndarray:
+        theta = self.angle_1
+        phi = self.angle_2
```

Comment:
Can be private, since users would be checking gate information in the SDK rather than in the default simulator internal classes.
```suggestion
        self._angle_1 = angle_1
        self._angle_2 = angle_2

    @property
    def _base_matrix(self) -> np.ndarray:
        theta = self._angle_1
        phi = self._angle_2
```